x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/action/TransportRunAnalyticsAction.java

```diff
@@ -38,9 +38,9 @@
 import org.elasticsearch.xpack.core.ml.action.RunAnalyticsAction;
 import org.elasticsearch.xpack.core.ml.utils.ExceptionsHelper;
 import org.elasticsearch.xpack.ml.MachineLearning;
-import org.elasticsearch.xpack.ml.analytics.DataFrameDataExtractorFactory;
-import org.elasticsearch.xpack.ml.analytics.DataFrameFields;
-import org.elasticsearch.xpack.ml.analytics.process.AnalyticsProcessManager;
+import org.elasticsearch.xpack.ml.dataframe.DataFrameDataExtractorFactory;
+import org.elasticsearch.xpack.ml.dataframe.DataFrameFields;
```

Comment:
the only downside I see: I have a class `DataFrameField` in the data frame plugin. When working with an IDE it's easy to pick the wrong one. But that's also true for the HLRC, anyway, if you see a chance for a better name, that would be great. 

(off-topic comment, the classname hasn't changed in this PR)